go-controller/pkg/factory/factory.go

```diff
@@ -854,10 +897,18 @@ func (wf *WatchFactory) ServiceInformer() cache.SharedIndexInformer {
 	return wf.informers[ServiceType].inf
 }
 
+func (wf *WatchFactory) EndpointSliceInformer() cache.SharedIndexInformer {
```

Comment:
this change has nothing to do with egressSVC ? why its here

Reply:
it shows up in the node controller